File: flight-booking/server/src/FlightBooking.API/Identity/HttpModels/RegisterRequest.cs
using FlightBooking.Business.Entities;

namespace FlightBooking.API.Identity.HttpModels
{
    public class RegisterRequest
    {
        public User User { get; set; }
        public string Email { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
    }
}

File: accommodation-booking/client/accommodation-booking/src/app/modules/auth/models/register-request.ts
import { Address } from "./address.model";
import { UserType } from "./user.model";

export class RegisterRequest {
    username: string = '';
    email: string = '';
    password: string = '';
    firstName: string = '';
    lastName: string = '';
    address: Address = new Address();
    userType: UserType = UserType.HOST;
}